src/nvim/testdir/runtest.vim

```diff
@@ -64,7 +64,7 @@ set nomore
 lang mess C
 
 " Always use forward slashes.
-" set shellslash
+set shellslash
```

Comment:
I'd hold back on this because oldtests failures changed and it affects `shellescape` for cmd.exe but oldtests don't fail the Appevyor builds.

Reply:
It is possible to change an existing test if the specification changes or if there is a problem with the test itself.

The changes that commented out `set shellslash` in https://github.com/neovim/neovim/commit/79a8d905ab8908362109284240d2f9a75564f8ad seem correct at first glance, but that is a mistaken change, because vim uses `shellslash` if `cmd.exe` is the shell(In fact, we commented this out, so in case of `set shellslash`, the problem that the file expansion is wrong is missed). Thus, the change to re-enable `set shellslash` is the correct change.

If you want test to succeed, you should change it to run with `set noshellslash` with the following comment for each individual test that fails:

```vim
" FIXME: Because the test fails by issue #xxxx, set noshellslash and test execution
" FIXME: Run with set noshellslash because test fails with set shellslash
```